### PR TITLE
In this features the about and index page are done.

### PR DESCRIPTION
**Pull request by Dagic-zewdu**
- what I want to develop is a software company website(Matrix tech). you can look at this short video for a clear understanding https://www.loom.com/share/28e30c59492541c08cdaad1e542e2f43
- created index.html file which serves as the landing page of this project
-created about, HTML file which is a server for the about page
-All the styles, images, and javascript files are stored in the assets folder

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                </li>
+                <li class="menu-item">
+                    <a href="#">Contacts</a>
+                </li>
+                <li class="compaign">
+                    <button type="button" class="btn-compaign">
+                        Partcipate on projects
+                    </button>
+                </li>
+            </ul>
```
Comment (brytebee) [inline index.html:59]: - [ ] **Required:** You have done an amazing job deploying your app using GitHub pages👏. However, you haven't accurately set up your links.
You should have a link to the `Home` page that navigates users back to the Home page when they are on the `About` page.

- [ ] **Required:** The above should be implemented for both screens. (Desktop & Mobile)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+        <div class="logo-footer">
+            <img src="assets/images/logo.png" alt="logo imag" class="footer-img">
+            <p class="footer-quote lato font-small font-lighter">
+                We build our computers the way we build our cities—over time, without a plan, on top of ruins
+            </p>
+        </div>
+        <p class="mx-10 lato ">
+            &copy; Metrix Tech Allright reserved (@2021)
+        </p>
+    </footer>
```
Comment (brytebee) [inline index.html]: - [ ] **Required:** Your footer looks really good, however, we can make some changes to make it look more similar to the Template.
- Your footer should maintain the margin of the middle section
- Kindly add the copyright to the other footer text.
- consider separating the footer and the combined text with some padding. 

Your Design | Behance Template
:---: | :---:
image | image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        <div class="logo-footer">
+            <img src="assets/images/logo.png" alt="logo imag" class="footer-img">
+            <p class="footer-quote lato font-small font-lighter">
+                We build our computers the way we build our cities—over time, without a plan, on top of ruins
+            </p>
+        </div>
+        <p class="mx-10 lato ">
+            &copy; Metrix Tech Allright reserved (@2021)
+        </p>
+    </footer>
```
Comment (brytebee) [inline about.html]: - [ ] **Required:** On this `About` page, the footer is required to have a dark background.

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+        <p class="text-3 font-light coocgoose">
+            Consider us your technical co-founders: a natural extension of your team. We learn your operation, inside
+            and out. By precisely understanding your vision, we tailor-make digital products that drive success in your
+            market.
+        </p>
+        <h2 class="text-4 lighter-cocogoose cocogoose">Next Zoom conference(Thu Feb 15/2021)</h2>
+        <h2 class="text-5 lighter-lato lato">
+            @Addis Ababa Ethiopia
+        </h2>
+    </section>
```
Comment (brytebee) [inline index.html]: - [ ] **Required:** Consider making the following changes to the top section
- Fix all console errors.
image

- [ ] You have correctly aligned all elements 👏. However, consider the following changes to have a similar design to the Template.

1. The should be a size difference between you `h1` and `h2`.
2. The `h1` text should have a background image. The `font-family: cocogoose`  as pointed out above is not loaded.
3. The font-weight makes it difficult to read, it is therefore not good for UX. 
4. The appropriate `font-family` and `font-weight` should be applied.

Your Design | Template
:---: | :---:
image | image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+            As your technical partners, we demystify tech and neatly sidestep risk, ensuring that you achieve your goals
+            on time and on budget. We also help you navigate strategy, branding, and how to tell your story.
+        </p>
+        <p class="font-medium">
+            please contact us trough email for further questions and how to make your software applicable
+        </p>
+        <p class="font-bold font-medium">
+            metrixtech3@gmail.com
+        </p>
+    </section>
```
Comment (brytebee) [inline about.html]: - [ ] **Required:** Kindly fix the console errors on this page as well.

image

- [ ] **Required:** To have a similar design, consider the following:

1. The gap (padding/margin) given between the top of the page and the header is too big.
2. Use appropriate `font-family` and `font-weight`. Align the header to the centre on all screens of the `About` page.
3. The text isn't visible.
4. The mail is **_underlined_** and has a small arrowhead icon.

Your Design | Template
:---: | :---:
image | image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Dagic-zewdu 👋, 

## Highlights
Good job so far!
- good video description👏
- excellent commit messages👏
- no linter errors👏
- the app is deployed using GitHub pages👏

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @Dagic-zewdu , 

### Great work on making the changes requested by a previous reviewer 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [x] Please make sure that, as the previous reviewer requested, the footer matches the given template. The layout should be divided into two sections, with the logo on the left and the text on the right. Currently, you have the sections stacked on top of each other.
Also, please change the footer on the about page to have a dark background, as requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Dagic-zewdu** [comment]
The about page dark footer is not necessary(not described on the template) and my previous commenter doesn't give me suggestions on that so will stick to the white background footer.

File: about.html (hunk 3 above)
Comment (ReshyResh) [reply about.html]: @Dagic-zewdu you can see the previous reviewer requested the change of the background here.

**Comment by ReshyResh** [comment]
@Dagic-zewdu  it is a requirement from the template, if you open it you'll see that the background of the footer changes.


| Homepage footer  | About page footer |
| ------------- | ------------- |
|   |   |

**Comment by Dagic-zewdu** [comment]
Okay thanks i will work on that

**Review by ReshyResh — APPROVED** [review]
# Hi again, @Dagic-zewdu , 

### Good job addressing the changes requested previously.
#### Your capstone project is complete and it looks amazing!  There is nothing else to say other than... it's time to merge it!
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**